packages/core/src/polyfills/form-request-submit.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Javan Makhmali
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+;(function (prototype) {
+  if (!prototype) return
+
+  if (typeof prototype.requestSubmit === 'function') return
+
+  prototype.requestSubmit = function (submitter) {
+    if (submitter) {
+      validateSubmitter(submitter, this)
+      submitter.click()
+    } else {
+      submitter = document.createElement('input')
+      submitter.type = 'submit'
+      submitter.hidden = true
+      this.appendChild(submitter)
+      submitter.click()
+      this.removeChild(submitter)
+    }
+  }
+
+  function validateSubmitter(submitter, form) {
```

Comment:
Question: even if it is a polyfill, looks a nice code to have unit tests. What do you think?

Reply:
@tassioFront It makes sense, I don't know if we will actually be able to test the polyfill in isolation since tests with Stencil only works with stencil-dom. 
I'll see if I can test it with the atom button.

Reply:
Ok @igorwessel good point too hahaha, in theory the tests in atom button guarantees this

Reply:
It looks perfect to me, folks, once the polyfill only makes sense for the atom button as you said. 🚀

Reply:
@tassioFront @felipefialho done in [62c4823](https://github.com/juntossomosmais/atomium/pull/431/commits/62c482304396fdd50a4d19f9252ed6d252cca751) 💯

Reply:
I can't resolve the conversation :/ kkkkk but it is approved!